test/e2e.js

```diff
@@ -193,4 +193,54 @@ testing.describe("end to end", function() {
             });
         });
     });
+    testing.describe("on select filter", function() {
+        testing.it("all filter does not affect list", function() {
+            helpers.navigateToSite();
+            helpers.addTodo("New todo item");
```

Comment:
Since adding the 3 same todos is in all of your tests, you could refactor to another helper function to cover this.